hieradata/integration.yaml

```diff
@@ -335,3 +335,5 @@ users::usernames:
   - tomridd
   - tomrussell
   - tomwhite
+
+trade_tariff_uri: 'https://tariff-frontend-dev.cloudapps.digital/'
```

Comment:
This could probably go in `common.yaml` and just have overrides in Staging and Production as it would be useful to set this default on dev or local VMs, too.